plugins/permanentpen.md

```diff
@@ -27,15 +27,20 @@ italic: false,
 underline: false,
 strikethrough: false
 ```
+
 For more information on {{site.productname}} formats, refer to the [formats]({{site.baseurl}}/configure/content-formatting/#formats) section.
 
+## Live example
```

Comment:
Another one where this should maybe be `Try our <PLUGIN> demo`?

Reply:
Fixed in https://github.com/tinymce/tinymce-docs/pull/1645/commits/88472197906330b72ce1e2bc0c9924df2384d3f2